src/pages/home/ReportScreen.js

```diff
@@ -202,8 +202,9 @@ class ReportScreen extends React.Component {
         }
 
         const reportID = getReportID(this.props.route);
-        const addWorkspaceRoomPendingAction = lodashGet(this.props.report, 'pendingFields.addWorkspaceRoom');
-        const addWorkspaceRoomErrors = lodashGet(this.props.report, 'errorFields.addWorkspaceRoom');
+        const pendingAction = lodashGet(this.props.report, 'pendingFields.addWorkspaceRoom') || lodashGet(this.props.report, 'pendingFields.createChat');
+        const errors = lodashGet(this.props.report, 'errorFields.addWorkspaceRoom') || lodashGet(this.props.report, 'errorFields.createChat');
```

Comment:
I don't think this works because it's not a boolean, it's an object. We need to get an object containing all these errors.

Reply:
I added a comment explaining this a bit, but I don't think this is actually the case since this is just going to be one error or the other, it can't be both.

Reply:
Oh yeah you're right. It will be one or the other. My bad.